README.md

```diff
@@ -62,6 +62,8 @@ var map = new mapboxgl.Map({
 });
 ```
 
+Also, add to your site the Mapbox GL JS CSS in `mapbox-gl/dist/mapbox-gl.css`.
```

Comment:
Should we say `node_modules/mapbox-gl/dist/mapbox-gl.css`?  I think being explicit about `node_modules` may be helpful for users who aren't very experienced using node/npm.